### ISSUE TITLE
suggection for prefixing the father title to the block

### ISSUE THREAD

**Issue by wenjing100** [debanjandhar12/logseq-anki-sync#55]
I've got a little trouble when I start a subject e.g. "ABC", and my logseq log would be as follow:
![image](https://user-images.githubusercontent.com/55542032/158728910-0301aba8-736f-4c0e-8c4c-37f0f5111066.png)

sometimes I would learn a child item specificallly, but its name has been simplified like 'type'. If I just attach a trailing tag '#flashcard', then the anki card would only show  a title 'type' . you know this would be an annoying circumstance. 

And I hope this tool would support optional setting(or other functional way) which allows us to prefix their parent title to the block.

**Comment by debanjandhar12** [comment]
Duplicate of #40.

**Comment by wenjing100** [comment]
> Wait, I might have misunderstood it. Can you rephrase that?

e.g.
![image](https://user-images.githubusercontent.com/55542032/163337037-2f617b39-1773-4278-a62e-73bb706a29b4.png)
and this anki card would be like:
![image](https://user-images.githubusercontent.com/55542032/163337211-8ab83362-8836-4ead-8c0e-21b8aa936afb.png)
![image](https://user-images.githubusercontent.com/55542032/163337249-fe0a3dd2-9c70-4f42-853e-704f79966a06.png)
------------------------------------
I holp the title on anki card would be like " knowledge of dogs -------type".

**Comment by debanjandhar12** [comment]
Seems like the same workflow can be achieved with #40.
